client/systems.go

```diff
@@ -220,7 +226,8 @@ type InstallSystemOptions struct {
 
 	// OnVolumes is the volume description of the volumes that the
 	// given step should operate on.
-	OnVolumes map[string]*gadget.Volume `json:"on-volumes,omitempty"`
+	OnVolumes       map[string]*gadget.Volume `json:"on-volumes,omitempty"`
+	OptionalInstall availableForInstall       `json:"optional-install,omitempty"`
```

Comment:
we probably need a way to distinguish empty vs not provided, so maybe needs to be a pointer?

Reply:
Is that because we will need to continue installing all optional (and present in the seed) things in the model, in the absence of an `optional-install` field?